stdlib/logging/__init__.pyi

```diff
@@ -31,13 +31,22 @@ class Filterer(object):
     def removeFilter(self, filter: _FilterType) -> None: ...
     def filter(self, record: LogRecord) -> bool: ...
 
+class Manager(object):
+    root: RootLogger
+    disable: int
+    loggerDict: dict[str, Logger]
+    def __init__(self, rootnode: RootLogger) -> None: ...
+    def getLogger(self, name: str) -> None: ...
```

Comment:
Oops, I had forgotten the correct way to handle missing attributes, documented in CONTRIBUTING.md:

```suggestion
    def getLogger(self, name: str) -> None: ...
    def __getattr__(self, name: str) -> Any: ...  # incomplete
```

Reply:
I can go ahead and fill out the rest of the public methods if needed.